css-layout-classes/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <header>
+            <h1>Choose your character!</h1>
+          </header>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
+          <article>
+            <img class="character" src="/images/link.png" alt="link" />
+          </article>
+        </div>
```

Comment:
You don't need this `article` element here. This likely works if `article` is a block element which takes up the full width of it's container, but it's an extra step to have an your image be 100% width of a block element which is 100% width of it's parent element that is 25% width of it's parent. Just let the `img` be a direct child of your `div.column-one-third`